inception-project/inception
# #3437 - Access to annotation page for curators

**What's in the PR**
- Enable access to annotation page to users that are only curators
- Add explicit session control to curation sidebar
- Added state badges to the curation and active learning sidebar to indicate when a session is in progress
- Cleaned up curation siderbar
- Updated documentation on curation sidebar

**How to test manually**
* See issue description - basically try the curation sidebar in different constellations as manager, curator, annotator

**Automatic testing**

**Documentation**
